pkg/api/api.go

```diff
@@ -122,7 +124,27 @@ func (dvol *DvolAPI) setActiveVolume(volumeName string) error {
 	}
 	defer file.Close()
 	encoder := json.NewEncoder(file)
-	encoder.Encode(currentVolumeContent)
+	if err := encoder.Encode(currentVolumeContent); err != nil {
+		return err
+	}
+	return nil
+}
+
+func (dvol *DvolAPI) setActiveBranch(volumeName, branchName string) error {
+	volume := dvol.dl.VolumeFromName(volumeName)
+	currentBranchJsonPath := filepath.FromSlash(volume.Path + "/current_branch.json")
+	currentBranchContent := map[string]string{
+		"current_branch": branchName,
+	}
+	file, err := os.Create(currentBranchJsonPath)
```

Comment:
I was thinking of what would happen if there was more information in the file but since that isn't the case right now this is fine.